src/NServiceBus.AcceptanceTests/MessageId/When_message_has_empty_id_header.cs

```diff
@@ -15,9 +15,9 @@ public class When_message_has_empty_id_header : NServiceBusAcceptanceTest
         public async Task A_message_id_is_generated_by_the_transport_layer()
         {
             var context = await Scenario.Define<Context>()
-                    .WithEndpoint<Endpoint>(g => g.When(async b => await b.SendLocal(new Message())))
-                    .Done(c => c.MessageReceived)
-                    .Run();
+                .WithEndpoint<Endpoint>(g => g.When(async b => await b.SendLocal(new Message())))
```

Comment:
Just realized that we could use the shorthand syntax without `async` and `await`